lib/rubocop/cop/rails/http_positional_arguments.rb

```diff
@@ -0,0 +1,89 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop is used to identify usages of `http methods
+      # like get,post,put,path` without the usage of keyword arguments
+      # in your tests and change them to use keyword arguments.
+      #
+      # @example
+      #   # bad
+      #   get :new, { user_id: 1}
+      #
+      #   # good
+      #   get :new, params: { user_id: 1 }
+      class HttpPositionalArguments < Cop
+        MSG = 'Use `keyword arguments` instead of ' \
+              'positional arguments for http call: '.freeze
+        KEY_WORD_ARGS = [:headers, :env, :params, :body, :flash, :as].freeze
+        HTTP_METHODS = [:get, :post, :put, :patch, :delete, :head].freeze
+
+        def on_send(node)
+          receiver, http_method, controller_node, data = *node
```

Comment:
I sacrificed consistency for readability and documentation purposes.  Which is more important?  Also please document what you want in a contributing doc to save others time.